tests/mockgun/schema.pickle

```diff
@@ -98681,4 +98681,4 @@ I01
 sS'value'
 p48535
 g47
-ssss.
\ No newline at end of file
+ssss.
```

Comment:
Not sure what happened there, but the pre-commit hooks are reporting this as something that needs to be fixed now.